common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/checksum/ShuffleChecksumHelper.java

```diff
@@ -153,6 +152,8 @@ public static Cause diagnoseCorruption(
       logger.warn("Unable to diagnose shuffle block corruption", e);
       cause = Cause.UNKNOWN_ISSUE;
     }
+    logger.info("Shuffle corruption diagnosis took {} ms, checksum file {}, cause {}",
+            duration, checksumFile.getAbsolutePath(), cause);
     return cause;
```

Comment:
This is not logged in the upper layer?

Reply:
This is part of the FetchFailedException's diagnosis reason - and shows up there.

I am not strongly for or against this ...
+CC @Ngone51

Reply:
Oh, if it exists, yes, we don't need this.

Reply:
It's up there but the executor logs. So maybe fine to log it too in shuffle service logs.

Reply:
Got it~